### PR TITLE
Update unit tests / Add CI 

### PR DESCRIPTION
**Pull request by wesdrew**
Using unittest modules test discovery feature to run unit tests. This should make it easier to add tests to project; unittest will take care of loading any tests and we won't have to manually add them to run_tests.py. 

Tests can be run by executing shell script 'run_tests.sh'.

Add a markup file for Travis CI and a badge in the project README to display the status of the unit test script. Trying to add a second service, Coveralls, to display code coverage but its not configured properly yet.

The CI service might require someone with ownership access to register the project repo.

### CODE REVIEW

File: mycity/platforms/amazon/models/en_US.json [hunk 1]
```diff
@@ -175,7 +175,16 @@
                         "value": "What's your address?"
                     }
                 ]
-            }
+            },
+	    {			// testing whether we can return a response with more information
+		"id": "Elicit.Slot.621270469710.323849903547", // + 1 to Elict.Slot.id above
+		"variations": [
```
Comment (jmartini) [inline mycity/platforms/amazon/models/en_US.json]: This looks unrelated

**Comment by jmartini** [comment]
Also please update documentation to explain how to run tests now

File: mycity/platforms/amazon/models/en_US.json (hunk 1 above)
Comment (wesdrew) [reply mycity/platforms/amazon/models/en_US.json]: Yeah you want me to remove it?

Comment (wesdrew) [reply mycity/platforms/amazon/models/en_US.json]: Done

**Comment by wesdrew** [comment]
AWAY WE GO!
